.golangci.yml

```diff
@@ -10,11 +10,21 @@ linters:
     - nlreturn
     - wsl
     - varnamelen
-    - scopelint # deprecated
-    - golint # deprecated
-    - gomoddirectives # deprecated
-    - interfacer # deprecated
-    - maligned # deprecated
+    # Deprecated
+    - scopelint
```

Comment:
just explain me again why we disable deprecated linters ?

Reply:
Because these ones are deprecated by the owner - and because they produce annoying warning output messages... just less noise on screen